<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CCMessenger</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Widget{
border-image: url(:/res/bluebackground.jpg);
border-width : 1px;

border-color : #1ea4f2;

border-style : solid;

border-width : 1px;
}


*{
font-size:17px;
}

#frame{border-radius:15px;}

</string>
  </property>
  <widget class="QTextBrowser" name="messageBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>751</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="messageTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>751</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="userTableWidget">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>10</y>
     <width>421</width>
     <height>741</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>731</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;宋体&quot;;</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>120</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>39</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>38</number>
   </attribute>
   <column>
    <property name="text">
     <string>用户名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>主机名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>IP地址</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>760</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);
border-radius:5px;
color: rgb(255, 255, 255);
font: 16pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>760</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);
font: 16pt &quot;宋体&quot;;
border-radius:5px;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="userlabel">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>760</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>在线用户：0人</string>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>551</y>
     <width>154</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>550</y>
     <width>313</width>
     <height>38</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="sizeComboBox">
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>24</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>26</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>27</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>28</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>29</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="colorToolButton_2">
      <property name="toolTip">
       <string>字体颜色</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/color3.jpg</normaloff>:/res/color3.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="sendToolButton">
      <property name="toolTip">
       <string>文件传输</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/document.jpg</normaloff>:/res/document.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="saveToolButton">
      <property name="toolTip">
       <string>保存聊天记录</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/save.jpg</normaloff>:/res/save.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="searchToolButton">
      <property name="toolTip">
       <string>查看聊天记录</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>res/document2.png</normaloff>res/document2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="clearToolButton">
      <property name="toolTip">
       <string>清空消息</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/clear.jpg</normaloff>:/res/clear.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>messageBrowser</zorder>
  <zorder>messageTextEdit</zorder>
  <zorder>userTableWidget</zorder>
  <zorder>sendButton</zorder>
  <zorder>exitButton</zorder>
  <zorder>userlabel</zorder>
  <zorder>fontComboBox</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>searchToolButton</zorder>
  <zorder>clearToolButton</zorder>
  <zorder>clearToolButton</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
